lib/cfd_trading/cfd_order_detail.dart

```diff
@@ -116,12 +138,21 @@ class _CfdOrderDetailState extends State<CfdOrderDetail> {
                                   onPressed: () async {
                                     try {
                                       await api.settleCfd(takerAmount: 40000, makerAmount: 20000);
+                                      FLog.info(text: "Successfully settled cfd.");
+
+                                      // refreshing cfd list after cfd has been closed
+                                      await cfdTradingService.refreshCfdList();
+
+                                      context.go(CfdTrading.route);
```

Comment:
🛠️ We trigger the router in the code below on line 166 - I don't think we should jump here, because that would mean that we introduce a "flicker" because we route here, then change index and route again 🙈

Reply:
Seems legit.